src/analysis/ODE/picard_lindelof.lean

```diff
@@ -49,6 +54,20 @@ structure picard_lindelof (E : Type*) [normed_add_comm_group E] [normed_space 
 (norm_le' : ∀ (t ∈ Icc t_min t_max) (x ∈ closed_ball x₀ R), ∥to_fun t x∥ ≤ C)
 (C_mul_le_R : (C : ℝ) * max (t_max - t₀) (t₀ - t_min) ≤ R)
 
+/-- `Prop` structure holding the hypotheses of the Picard-Lindelöf theorem.
+
+The similarly named `picard_lindelof` structure is part of the internal API for convenience, so as
+not to constantly invoke choice, but is not intended for public use. -/
+structure is_picard_lindelof
```

Comment:
This looks good now.

How about using it in the old `picard_lindelof` structure like this:
```lean
structure picard_lindelof (E : Type*) [normed_add_comm_group E] [normed_space ℝ E] :=
(to_fun : ℝ → E → E)
(t_min t_max : ℝ)
(t₀ : Icc t_min t_max)
(x₀ : E)
(C R L : ℝ≥0)
(is_pl : is_picard_lindelof to_fun t_min t₀ t_max x₀ L R C)
```
It shouldn't be too hard to fix the old code for this change. If so, then I think this is worth doing.